MQTicketService/Dockerfile

```diff
@@ -11,37 +11,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM openjdk:8-alpine
+FROM adoptopenjdk/openjdk11:alpine
```

Comment:
The application that runs inside the docker image is already maven enabled, so we should be using maven to package the executable and copy only the uber jar that gets created into the run image.

We will then not need all the wget statements as a mvn package clean will fetch all the dependencies.

The first image can be based on
https://hub.docker.com/_/maven
eg.
FROM maven:3.6-jdk-11
or
FROM 3.8.4-openjdk-11

see https://stackoverflow.com/questions/27767264/how-to-dockerize-maven-project-and-how-many-ways-to-accomplish-it

Reply:
Removed all wget statements and changed first image to maven:3.6-jdk-11 .